azps-1.8.0/Az.Accounts/Connect-AzAccount.md

```diff
@@ -537,8 +608,14 @@ Accept wildcard characters: False
 ```
 
 ### -Tenant
+
 Optional tenant name or ID
 
+> [!NOTE]
+> Due to limitations of the current API, a tenant ID must be specified instead of a tenant name when
+> signing in with a business-to-business (B2B) account that has been assigned access to your Azure
+> tenant.
```

Comment:
Passive voice

```suggestion
> Due to limitations of the current API, you must use a tenant ID instead of a tenant name when
> signing in with a business-to-business (B2B) account that has been assigned access to your Azure
> tenant.
```